packages/browser-agent-core/common/window/session-storage.js

```diff
@@ -0,0 +1,28 @@
+/**
+ * Interface for Agents' usage of the Window.sessionStorage as a replacement to third-party cookies.
+ *  (All agents on a tab|window will share the same sessionStorage.)
+ * 
+ * @documentation https://newrelic.atlassian.net/wiki/spaces/INST/pages/2522513791/JSESSIONID+Cookie+Change+Design+Document
+ * @environment Browser script
+ */
+import { generateRandomHexString } from '../ids/unique-id'
+
+export { getCurrentSessionIdOrMakeNew };
+
+const ss = window.sessionStorage;
+const SESS_ID = Symbol("session id").toString(); // prevents potential key collisions in session storage
```

Comment:
Symbol is not supported on all browsers

Reply:
Good call out. Since symbols would increase the polyfill from 62kb to 82kb, it's deemed unnecessary, and this is usage is removed.